File: framework/cei_python3/impl/restfuloptions.py


class RestfulOptions(object):
    url = None
    connection_timeout = None
    api_key = None
    secret_key = None

    def set_from(self, options):
        if options.url is not None:
            self.url = options.url
        if options.connection_timeout is not None:
            self.connection_timeout = options.connection_timeout
        if options.api_key is not None:
            self.api_key = options.api_key
        if options.secret_key is not None:
            self.secret_key = options.secret_key

File: framework/cei_java/src/main/java/cn/ma/cei/impl/RestfulOptions.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package cn.ma.cei.impl;

/**
 *
 * @author u0151316
 */
public class RestfulOptions {
    public String url = null;
    public Integer connectionTimeout = null;
    public String apiKey = null;
    public String secretKey = null;
    
    public void setFrom(RestfulOptions options) {
        if (options.url != null) {
            this.url = options.url;
        }
        if (options.connectionTimeout != null) {
            this.connectionTimeout = options.connectionTimeout;
        }
        if (options.apiKey != null) {
            this.apiKey = options.apiKey;
        }
        if (options.secretKey != null) {
            this.secretKey = options.secretKey;
        }
    }
}
